v5/ooapi.v5.core/CoreStartup.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using ooapi.v5.core.Services.Interfaces;
+using ooapi.v5.core.Services;
+
+namespace ooapi.v5.core;
+
+/// <summary>
+/// Startup
+/// </summary>
+public static class CoreStartup
+{
+    /// <summary>
+    /// Method used adding services to dependency injection
+    /// </summary>
+    /// <param name="servicesCollection">ServiceCollection used for dependency injection</param>
+    public static void Startup(IServiceCollection servicesCollection)
+    {
+        servicesCollection.AddServices();
+    }
+
+    private static IServiceCollection AddServices(this IServiceCollection builder)
+    {
+        builder.AddScoped<IAcademicSessionsService, AcademicSessionsService>();
```

Comment:
```suggestion
        builder.TryAddScoped<IAcademicSessionsService, AcademicSessionsService>();
```

en uiteraard ook voor de regels hieronder

Reply:
Waarom? In principe wordt die startup maar 1x aangeroepen, als je dan dingen dubbel in de servicecollection zet krijg je tenminste een fout.

Reply:
TryAdd laat het toe om eventueel een bepaalde implementatie te overschrijven als je dat zou willen. ( Dus als je een registratie doet voor de AddServices Extension method call ). Je bent dus wat flexibeler.